blocks/components/editable/tinymce.js

```diff
@@ -1,5 +1,5 @@
 export default class TinyMCE extends wp.element.Component {
-	componentDidMount() {
+	initialize() {
```

Comment:
Might want to consider conventions for component method ordering. I usually like to see:

- Static properties
- Constructor
- Lifecycle methods
- Miscellaneous
- Render

(Considering this as "miscellaneous")